src/main/java/org/spongepowered/common/mixin/core/entity/item/ArmorStandEntityMixin.java

```diff
@@ -64,49 +64,49 @@ private void fireDestroyDamageEvent(final DamageSource source, final CallbackInf
                 cir.setReturnValue(false);
             }
             if (event.getFinalDamage() < this.getHealth()) {
-                this.damageArmorStand((float) event.getFinalDamage());
+                this.shadow$func_213817_e(source, (float) event.getFinalDamage());
                 cir.setReturnValue(false);
             }
         }
     }
 
     @Inject(method = "attackEntityFrom",
             slice = @Slice(from = @At(value = "FIELD", target = "Lnet/minecraft/util/DamageSource;OUT_OF_WORLD:Lnet/minecraft/util/DamageSource;")),
-            at = @At(value = "INVOKE", target = "Lnet/minecraft/entity/item/EntityArmorStand;setDead()V", ordinal = 0),
+            at = @At(value = "INVOKE", target = "net/minecraft/entity/item/ArmorStandEntity.remove()V", ordinal = 0),
```

Comment:
Don't change from `Lclass/path;method()return`, keep it prefixed with `L` and split the method with `;`

Reply:
Well that atleast explains why MC dev was confused. Copying from bytecode a little too liberally

Reply:
It's supported to use non `L` prefixed syntax, but you would need to change `/` to `.` from what I remember.

Reply:
It's also why we chose to stick with `L` format, because it's abundantly clear where the class definition ends and method/field name begin.